src/core/EntryAttachments.cpp

```diff
@@ -37,7 +37,11 @@ bool EntryAttachments::hasKey(const QString& key) const
 
 QSet<QByteArray> EntryAttachments::values() const
```

Comment:
I actually question the validity of this function in general. It gives up the purpose of the EntryAttachments class. In fact, its only used in one place [Entry.cpp:L579](https://github.com/keepassxreboot/keepassxc/blob/82bf00de7389db3e52baf11c9295300ad943c643/src/core/Entry.cpp#L579) in a rather useless fashion!